sdk/src/test/java/io/dapr/runtime/DaprRuntimeTest.java

```diff
@@ -189,4 +280,46 @@ private static final String generateMessageId() {
   private static final Map<String, String> generateSingleMetadata() {
     return Collections.singletonMap(UUID.randomUUID().toString(), UUID.randomUUID().toString());
   }
+
+  private static final class Message {

```

Comment:
Does this correspond to CloudEventsEnvelope?  

https://github.com/dapr/components-contrib/blob/ede980c7ff320bc38f3284d12793910f4f4f4d93/pubsub/envelope.go#L23

It doesn't seem to have all the fields.

Reply:
No. This is just for this test. It is all the input we need to test a message.

Reply:
I got your comment. It is about the other Message class (not private). I will rename it and make it part of the domain for our API. I will also add the other attributes. It is more extensible that way.

Reply:
Right I pointed at the wrong Message class above, meant the other one that it not test-only.